src/modules/ipfs/nft.storage/index.ts

```diff
@@ -0,0 +1,12 @@
+export default function uploadFile(path: string) {
+  return fetch("https://api.nft.storage/upload", {
+    //@ts-ignore
+    body: fs.createReadStream(path),
+    method: "POST",
+    headers: {
+      "Authorization": `Bearer ${process.env.NFT_STORAGE_API_KEY || ''}`,
+      "Content-Type": "application/x-www-form-urlencoded",
+    } 
+  })
```

Comment:
Yeah, this is the spot to do the await on the json. Have it return ready-to-go file responses.